bluesky/examples.py

```diff
@@ -473,7 +473,8 @@ def stage(self):
         self._file_stem = short_uid()
         self._path_stem = os.path.join(self.save_path, self._file_stem)
```

Comment:
I agree that this could be factored a little differently, but I wanted to give the author some space to breathe. Meanwhile, what about the diff in this PR? It is a prerequisite to getting export tests working.